core/src/mindustry/type/StatusEffect.java

```diff
@@ -15,10 +15,12 @@
     public float damageMultiplier = 1f;
     /** Unit health multiplier. */
     public float healthMultiplier = 1f;
-    /** Unit speed multiplier */
+    /** Unit speed multiplier. */
     public float speedMultiplier = 1f;
```

Comment:
hmm, maybe rename it to `moveSpeedMultiplier` to make it a little more descriptive, but then again it might break modding compatibility, and rules.java has borked speed rules anyways, i don't know tbh 🤔

Reply:
I'm not qualified to make that call, I'll leave it to Anuke.